pkg/log/handler_sentry_context.go

```diff
@@ -18,8 +20,7 @@ func SentryContextConfigProvider(config ConfigProvider, handler *HandlerSentry)
 }
 
 func SentryContextEcsMetadataProvider(_ ConfigProvider, handler *HandlerSentry) error {
-	ecsMetadata, err := ReadEcsMetadata()
-
+	ecsMetadata, err := ecs.ReadEcsMetadata()
```

Comment:
If the package is already called `ecs`, the method could just be called `ReadMetadata`